packages/nocodb/src/lib/meta/api/userApi/ui/auth/resetPassword.ts

```diff
@@ -86,7 +86,7 @@ export default `<!DOCTYPE html>
             });
             this.success = true;
           } catch (e) {
-            alert('Some error occured')
+            alert('Failed to reset password: ' + e.response.data.msg)
```

Comment:
Exception here will only be in the form of `e.response.data` r8? Otherwise might cause issues